mappings/net/minecraft/block/Block.mapping

```diff
@@ -1,6 +1,6 @@
 CLASS none/akw net/minecraft/block/Block
 	FIELD A stateFactory Lnone/ask;
-	FIELD a DEFAULT_ID Lnone/kp;
+	FIELD a AIR_ID Lnone/kp;
```

Comment:
Minecraft uses the field as the default "this block doesn't exist in the registry!" ID. Is it used anywhere else to faciliate the name change?